build.gradle

```diff
@@ -8,13 +9,16 @@ repositories {
 }
 
 buildscript {
+    ext.kotlin_version = '1.2.31'
     repositories {
         maven {
             url "https://plugins.gradle.org/m2/"
         }
+        mavenCentral()
     }
     dependencies {
         classpath "me.champeau.gradle:jmh-gradle-plugin:0.3.1"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
```

Comment:
I'm excited to integrate some Kotlin code into the Android half of the codbase, but I don't think it's worth the value tradeoff today to integrate kotlin code into the core library until we have a bit more experience and direction.